src/test/java/com/revature/services/UserServiceTest.java

```diff
@@ -64,6 +79,18 @@ void testFindById() {
 		assertEquals(expected, actual);
 		verify(this.mockUserRepo, times(1)).findById(id);
 	}
+	
+	@Test
+	void testFindById_Failure_UserNotFound() {
+		int id = 1337;
+		given(this.mockUserRepo.findById(id)).willReturn(Optional.empty());
+		try {
+			this.uServ.findById(id);
+			fail("Expected an exception to be thrown");
```

Comment:
You should specify which exception type is expected to be thrown. In this case, it's `UserNotFoundException`